src/config/axiosAuthMiddleware.js

```diff
@@ -1,13 +1,13 @@
-import axios from 'axios';
+import api from './api';
 
 function axiosAuthMiddleware() {
   return ({ getState }) => next => (action) => {
```

Comment:
Which notation? We configured our own axios instance in `api.js`, see above.
Instead of applying configuration to axios itself we will apply it to our instance. 
So we are importing api instead of axios. 
It's a little bit of magic actually that we can add config to the instance by importing it like this but it works I don't know.

Reply:
`({ getState }) => next => (action) => {function body}`
It looks like an arrow function on the return value of an arrow function on the return value of an arrow function? Not any problem, I just don't understand the line.